app/models/manageiq/providers/cloud_manager/provision_workflow.rb

```diff
@@ -26,9 +26,18 @@ def allowed_cloud_subnets(_options = {})
   end
 
   def allowed_cloud_networks(_options = {})
+    return {} unless (src_obj = provider_or_tenant_object)
+
     source = load_ar_obj(get_source_vm)
     targets = get_targets_for_source(source, :cloud_filter, CloudNetwork, 'cloud_network_id')
-    allowed_ci(:cloud_network, [:availability_zone], targets.map(&:id))
+
+    if src_obj.all_cloud_networks.nil?
```

Comment:
The `all_cloud_networks` call should always return an array since it is an association ([here](https://github.com/ManageIQ/manageiq/blob/master/app/models/manageiq/providers/network_manager.rb#L14) and [here](https://github.com/ManageIQ/manageiq/blob/master/app/models/manageiq/providers/network_manager.rb#L50)) or a combination of two arrays for a [tenant](https://github.com/ManageIQ/manageiq/blob/master/app/models/cloud_tenant.rb#L119-L121).

Do you really want `.blank?` here?

Reply:
It wasn't always returning an array.

Reply:
Can you provide a reference to where it would not?  Is this just in a testing scenario?

Reply:
Just in testing.